Dockerfile

```diff
@@ -10,6 +10,7 @@ RUN set -ex \
                augeas-libs \
                libffi \
                libssl1.0 \
+               wget \
```

Comment:
Hi lsowen,
when do we need to wget https://blabla.com/ ?

gnu wget is about 470k, and busybox has a simple wget without ssl_helper.